templates/part.content.php

```diff
@@ -1,10 +1,39 @@
-<p>Hello World <?php p($_['user']) ?></p>
-
-<p><button id="hello">click me</button></p>
-
-<p><textarea id="echo-content">
-	Send this as ajax
-</textarea></p>
-<p><button id="echo">Send ajax request</button></p>
-
-Ajax response: <div id="echo-result"></div>
\ No newline at end of file
+<div class="section" id="systemSection">
+	<h2><?php p($l->t('System'));?></h2>
+	<p><?php p($l->t('CPU'));?></p>
+	<canvas id="cpuloadcanvas" width="400" height="100"></canvas>
+	<p><em><?php p($l->t('CPU load average (Last minute)')); ?></em></p>
+	<br>
+	<p><?php p($l->t('Memory'));?></p>
+	<canvas id="memorycanvas" width="400" height="100"></canvas>
+	<p><em id="memFooterInfo"></em></p>
+</div>
+<div class="section" id="storageSection">
+	<h2><?php p($l->t('Storage'));?></h2>
+	<p><?php p($l->t('Users: '));?><em id="numUsersStorage">--</em></p>
+	<p><?php p($l->t('Files: '));?><em id="numFilesStorage">--</em></p>
+	<p><?php p($l->t('Storages: '));?></p>
```

Comment:
Translators will move the empty space at the beginning depending on language. I'd move the space out here of the translatable string.